docs/UserGuide.md

```diff
@@ -51,15 +52,13 @@ not be so obvious!
 1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+1. Type the command in the command box and press `Enter` on your keyboard to execute it. e.g. typing **`help`** and pressing `Enter` will open the help window.<br>
```

Comment:
If you use `enter` here, does it mean it is a command? as in our user guide, content in `` is considered as a command entering by users

Reply:
Noted, thanks!